examples/common/scripts/vttablet-up.sh

```diff
@@ -29,9 +29,9 @@ tablet_hostname=''
 printf -v tablet_logfile 'vttablet_%010d_querylog.txt' $uid
 
 tablet_type=replica
-if [[ "${uid: -1}" -gt 1 ]]; then
- tablet_type=rdonly
-fi
+#if [[ "${uid: -1}" -gt 1 ]]; then
```

Comment:
This is for local testing purpose. I will remove this before final commit to the PR.